### PR TITLE
feat/storybook-7-upgrade-2

### PR DESCRIPTION
**Pull request by panvourtsis**

### CODE REVIEW

File: .storybook/main.ts [hunk 1]
```diff
@@ -21,71 +19,55 @@ function getPackageDir(filepath: string) {
...
+    '../docs/tokens/TOKENS.mdx',
+    '../docs/tokens/globals/*.mdx',
+    '../docs/tokens/*.mdx',
+    '../docs/system/*.mdx',
+    '../src/**/*.@(mdx|stories.@(ts|tsx))',
   ],
+
   addons: [
-    '@storybook/addon-actions/register',
+    // '@storybook/addon-actions/register',
```
Comment (kostasdano) [inline .storybook/main.ts:35]: unnecessary comment

File: .storybook/main.ts [hunk 2]
```diff
@@ -21,71 +19,55 @@ function getPackageDir(filepath: string) {
...
+    '@storybook/addon-interactions',
     '@storybook/addon-links',
     '@storybook/addon-viewport',
     '@storybook/addon-knobs',
-    '@storybook/addon-a11y',
-    '@storybook/addon-storysource/register',
+    '@storybook/addon-a11y', // '@storybook/addon-storysource/register',
```
Comment (kostasdano) [inline .storybook/main.ts:40]: unnecessary comment

File: .storybook/manager.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { addons } from '@storybook/manager-api';
+import { create } from '@storybook/theming';
+
+const theme = create({
+  base: 'light',
+  // brandImage: logo,
```
Comment (kostasdano) [inline .storybook/manager.ts:6]: is this needed?

Comment (panvourtsis) [reply .storybook/manager.ts:6]: is future reference for the logo of the tool

File: .storybook/main.ts (hunk 2 above)
Comment (panvourtsis) [reply .storybook/main.ts]: fixed here https://github.com/Orfium/orfium-ictinus/pull/731/commits/84a8ca0c4b44e474a25fec49a37d21c7645d7593

File: .storybook/main.ts (hunk 1 above)
Comment (panvourtsis) [reply .storybook/main.ts]: fixed here https://github.com/Orfium/orfium-ictinus/pull/731/commits/84a8ca0c4b44e474a25fec49a37d21c7645d7593

File: src/vite-env.d.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+/// <reference types="vite/client" />
+/// <reference types="vitest/globals" />
+/// <reference types="vite-plugin-svgr/client" />
+/// <reference types="./theme/emotion.d.ts" />
```
Comment (kostasdano) [inline src/vite-env.d.ts:4]: is this needed?

File: tsconfig.json [hunk 5]
```diff
@@ -1,48 +1,26 @@
...
-    "types": ["@emotion/react/types/css-prop", "node", "jest", "@testing-library/jest-dom"],
+    "noEmit": false,
+    "jsx": "react-jsx",
+    "module": "ESNext",
+    "types": ["@emotion/react/types/css-prop"],
+    "allowUnreachableCode": false,
+    /* Linting */
+    //    "strict": true,
+    //    "noUnusedLocals": true,
+    //    "noUnusedParameters": true,
```
Comment (kostasdano) [inline tsconfig.json:20]: comments

File: vite.config.ts [hunk 6]
```diff
@@ -0,0 +1,97 @@
+/// <reference types="vitest" />
```
Comment (kostasdano) [inline vite.config.ts:1]: comment

**Review by kostasdano — APPROVED** [review]

File: src/vite-env.d.ts (hunk 4 above)
Comment (panvourtsis) [reply src/vite-env.d.ts:4]: yes these are the references of vite

File: tsconfig.json (hunk 5 above)
Comment (panvourtsis) [reply tsconfig.json:20]: i suggest leaving them and reopening them when we finish these changes.

File: vite.config.ts (hunk 6 above)
Comment (panvourtsis) [reply vite.config.ts:1]: its a reference

**Pull request merged**